RPIPs/RPIP-bounty_standards.md

```diff
@@ -0,0 +1,119 @@
+* The payout of incentives for a given bounty MUST NOT reduce the bounty completion payout. 
+* When incentive levers are modified by the GMC, incentives SHOULD be paid out in such a way that ambiguity favours incentive recipients.
+* The total cost of lever incentives is not fixed. The GMC SHOULD set lever incentives with caution when funds are limited. 
+
+#### Payout
+* Bounty incentives SHOULD be paid out of the GMC budget.
+* Incentive payouts SHOULD take place within the GMC's regular payout schedule.
+* Incentive payouts MUST be denominated in dollars, and MUST be paid out in one of the GMC's accepted payout token options selected by the recipient.
+
+#### Lever A: Proposal Incentive
+The proposal incentive lever aims to influence the number of bounty proposals that are recieved by the GMC.
+* This incentive MUST be paid to the proposal author for a bounty proposal that is adopted by the GMC. 
+* Individual incentive payouts for this lever MAY be reduced at the GMC's discretion based on proposal quality and completeness.
+* Bounty definitions do not need to be included for a proposer to recieve this incentive.
+
+#### Lever B: Definition Incentive
+The definition incentive lever aims to influence the quality and completeness of bounty definitions accompanying bounty proposals.
+* This incentive MUST be paid to the definiton author for bounty definitions provided alongside a bounty proposal that is adopted by the GMC. 
+* This incentive payout SHOULD be paid a maximum once per proposal, even if the bounty proposal is split into multiple definitions.
+* Individual incentive payouts for this lever MAY be reduced at the GMC's discretion based on definition quality and completeness.
+* This incentive SHOULD go to the proposal author unless an alternative is explicitly preapproved by the GMC adminitrator.
+
+#### Lever C: Completion Incentive
+The completion incentive lever aims to influence the quality and consistency of bounty work submitted by bounty hunters. It aims to do this by rewarding contacts that support, oversee and assist bounty hunters meet the requirements of open bounties. 
+* This incentive SHOULD be split equally between bounty contacts that meet the following conditions:
+  1. The bounty MUST be completed to the satisfaction of the GMC.
+  2. The contact MUST be publicly listed in the bounty definition of the completed bounty.
+  3. The contact MUST NOT be affiliated with the completing bounty hunter.
+  4. The bounty hunter does not comment negatively on the contact when asked by the GMC.
+* Contacts SHOULD NOT assist the bounty hunter directly with the bounty work.
+* Contacts SHOULD assist bounty hunters via feedback, direction and oversight upon request.
+* Individual incentive payouts for this lever MAY be reduced at the GMC's discretion for any reason. Reasoning must be publicly stated by the GMC.
+
+#### Initial Lever Recommendations
```

Comment:
I like the recommendations, does it make sense to float some concrete numbers for these? I guess ideally the RPIP would be presented and it could be commented on separately in the thread for the GMC's consideration.

Reply:
Was hoping to avoid. Will see if there is any commentary.